org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportReplay.java

```diff
@@ -1242,6 +1271,66 @@ final private String getThisSymProperty(final String symbology,
     }
     return null;
   }
+  
+  private List<Editable> addElementsToExistingLayer(Layer layerToAddTo,Editable l)
```

Comment:
This method, and `injectContent` can both be made static.  

The advantage of making them static is we can be sure they don't rely on member state (member fields).